### ISSUE TITLE
admin api: change password is broken

### ISSUE THREAD

**Issue by loicjaouen** [dhlab-basel/Knora#1284]
from the docs:
https://github.com/dhlab-basel/Knora/blob/develop/docs/src/paradox/03-apis/api-admin/users.md#update-users-password

```
PUT: /admin/users/iri/<userIri>/Password
{
  "oldPassword": "test",
  "newPassword": "test1234"
}
```

should change the password.

What we get is a `400 Bad Request` response:
```
The request content was malformed:
Too few parameters sent for password change.
```

**Comment by gfoo** [comment]
```json
{
newPassword:
requesterPassword:
}
```

**Comment by loicjaouen** [comment]
thanks @gfoo ;)

when I put "requesterPassword" I get a `409 Conflict`:

```
{
    "error": "org.knora.webapi.UpdateNotPerformedException: User was not updated. Please report this as a possible bug."
}
```

**Comment by andreas-aeschlimann** [comment]
@loicjaouen for me it worked with the `requesterPassword` parameter. API docs should be updated though.

**Comment by subotic** [comment]
Yes, the documentation is wrong. There are these tests, that have the correct payload: 

https://github.com/dhlab-basel/Knora/blob/11c20080e433bb20e39f90ac036c6387c161ff99/webapi/src/test/scala/org/knora/webapi/e2e/admin/UsersADME2ESpec.scala#L268

https://github.com/dhlab-basel/Knora/blob/11c20080e433bb20e39f90ac036c6387c161ff99/webapi/src/test/scala/org/knora/webapi/e2e/admin/UsersADME2ESpec.scala#L290
